db/app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { handlePsqErrors, handleCustomErrors, handleServerErrors, handleBadPath } = require("./errors");
+const { getAllTopics } = require("./app.controller");
+
+const app =express()
+
+app.use(express.json())
+
+app.get('/api/topics', getAllTopics)
+
+
+
+
+app.all( (req,res,next) =>{
+    res.status(404).send("Not Found")
```

Comment:
Ideally you'd send this back in an object like `{msg:"Not Found"}` - or something like that